MapboxVision/src/main/java/com/mapbox/vision/utils/VisionLogger.kt

```diff
@@ -156,7 +156,7 @@ interface VisionLogger {
 
     private class FileImpl(private val minLogLevel: Int = Log.INFO) : VisionLogger {
 
-        val fileWriter = FileWriter("${Environment.getExternalStorageDirectory()}/vision.log")
+        val fileWriter = FileWriter("${Environment.getExternalStorageDirectory()}/vision-${System.currentTimeMillis()}.log")
```

Comment:
maybe write file with some DateTime format in name, not just long?

Reply:
Well, that makes sense - let's just move it out of this PR scope.